src/pages/User/User.jsx

```diff
@@ -55,6 +58,47 @@ export const User = () => {
             >
               <Avatar alt="User Profile" sx={{ width: 86, height: 86 }} />
               <h2>{otherUser?.data?.userName}</h2>
+              <Box
+                sx={{
+                  component: "div",
+                }}
```

Comment:
you can simply use the `as` attribute, like `as='div'` here instead of `sx`

Reply:
Okay got it